docs/jwst/assign_wcs/reference_files.rst

```diff
@@ -29,53 +29,132 @@ reftype                                     description
 
-The camera reference file contains an astropy compound model made up of a polynomial models, rotation and translations. The forward direction is from the FPA to the GWA.
 
-:model: Transform through the CAMERA.
+DISTORTION
+::::::::::
 
-COLLIMATOR
-----------
+For MIRI, DISTORTION reference files are currently selected based on the values of EXP_TYPE, DETECTOR, CHANNEL and BAND in the input science data set.
 
-CRDS Selection Criteria
-:::::::::::::::::::::::
+For FGS, DISTORTION reference files are currently selected based on the values of EXP_TYPE and DETECTOR in the input science data set.
 
-:NIRSPEC: EXP_TYPE
+For NIRCAM, DISTORTION reference files are currently selected based on the values of EXP_TYPE, DETECTOR, CHANNEL, and FILTER in the input science data set.
 
-Reference File Formats
-::::::::::::::::::::::
+For NIRISS, DISTORTION reference files are currently selected based only on the value of EXP_TYPE and PUPIL in the input science data set.
 
-This reference file contains an astropy compound model made up of a polynomial models, rotation and translations. The forward direction is from the GWA to the MSA.
 
-:model: Transform through the COLLIMATOR.
+FILTEROFFSET
+::::::::::::
+For MIRI, FILTEROFFSET reference files are currently selected based on the values of EXP_TYPE and DETECTOR in the input science data set.
 
-DISPERSER
----------
+FORE
+::::
 
-CRDS Selection Criteria
-:::::::::::::::::::::::
+For NIRSPEC, FORE reference files are currently selected based on the values of EXP_TYPE and FILTER in the input science data set.
+
+FPA
+:::
+For NIRSPEC, FPA reference files are currently selected based only on the value of EXP_TYPE in the input science data set.
+
+IFUFORE
+:::::::
+For NIRSPEC, IFUFORE reference files are currently selected based only on the value of EXP_TYPE in the input science data set.
+
+
+IFUPOST
+:::::::
+For NIRSPEC, IFUPOST reference files are currently selected based only on the value of EXP_TYPE in the input science data set.
+
+IFUSLICER
+:::::::::
+For NIRSPEC, IFUSLICER reference files are currently selected based only on the value of EXP_TYPE in the input science data set.
+
+
+MSA
+:::
+For NIRSPEC, MSA reference files are currently selected based only on the value of EXP_TYPE in the input science data set.
+
+OTE
+:::
+For NIRSPEC, OTE reference files are currently selected based only on the value of EXP_TYPE in the input science data set.
+
+SPECWCS
+:::::::
+For MIRI, SPECWCS reference files are currently selected based on the values of DETECTOR, CHANNEL, BAND, SUBARRAY, and EXP_TYPE in the input science data set.
+
+For NIRCAM, SPECWCS reference files are currently selected based on teh values of EXPT_TYPE, MODULE, and PUPIL in the input science data set.
+
+For NIRISS GRISM and TSGRIM modes, SPECWCS reference files are currently selected based on the values of SUBARRAY and EXP_TYPE in the input science data set.
```

Comment:
I think this is supposed to be NIRCam, not NIRISS, in which case it should read:
```
For NIRCam WFSS and TSGRISM modes, SPECWCS reference files are currently selected based on the values of EXP_TYPE, MODULE, and PUPIL in the input science data set.
```

Reply:
Can @nden or @sosey confirm this?

Reply:
nircam wfss  `specwcs` are based on EXP_TYPE, MODULE,  and PUPIL.  NIRISS wfss uses EXP_TYPE,  FILTER, and PUPIL. TSGRISM isn't in the NIRISS rmap at all. NRC (nircam) TSGRISM is selected the same way as the other nircam WFSS exposures (but it's only valid in module A)

I don't see it mentioned, but perhaps putting in a sentence somewhere in the docs about how to read the current rmap that tells you what is being selected on would be useful? If it's in the CRDS docs we can link back there.

Reply:
These are the crds docs: https://jwst-crds.stsci.edu/static/users_guide/rmap_syntax.html#reference-mappings-rmap, we should link there but also consider adding a short description somewhere that simplifies how to read them for users that just wonder what they mean.